data-pipeline/device-profile-updater/src/test/java/org/ekstep/ep/samza/service/DeviceProfileServiceTest.java

```diff
@@ -113,6 +154,7 @@ public void shouldNotUpdateFordidNull() throws Exception {
 
     @Test
     public void shouldNotUpdateFirstAccessifPresent() throws Exception {
+        stub(connectionMock.createStatement()).toReturn(statementMock);
```

Comment:
Rewrite the test case with embedded redis

Reply:
Done.